src/v/raft/consensus.h

```diff
@@ -222,6 +222,8 @@ class consensus {
     void suppress_heartbeats(model::node_id, follower_req_seq, bool);
 
 private:
+    static constexpr voter_priority max_voter_priority{100};
+    static constexpr auto target_priority_update_factor = 0.8;
```

Comment:
interesting.

1. we don't use auto for doubles/float - be explicit on these types. 
2. can we use integer math instead of floating point math.